contents/product-engineers/dogfooding.md

```diff
@@ -0,0 +1,131 @@
+
+![Data warehouse feedback](https://res.cloudinary.com/dmukukwp6/image/upload/me_7c76189301.png)
+
+## The benefits of dogfooding
+
+We dogfood a lot, but what do we actually get out of it?
+
+- **Fast feedback.** Getting feedback from your teammates is a lot easier and faster than getting it from users. This guides product development and helps you iterate rapidly.
+
+- **Reveals issues and bugs.** When we dogfood, we almost always discover usability issues or bugs that we can fix before they impact user experience. This helps you ship a polished product.
+
+- **Increases product knowledge.** Our whole team uses PostHog. This helps us fix cross-functional issues, discover use cases, and promote PostHog better.
+
+- **Spirit.** If your product is good, it helps your team become confident in it. If your product is bad, it motivates the team to make it better. As a dramatic example of this, Apple banned typewriters. They were either going to manifest the future of computing or die trying.
+
+![Apple](https://res.cloudinary.com/dmukukwp6/image/upload/apple_fd647f303d.png)
+
+## How to dogfood well
+
+From our experiences with dogfooding, we've learned a bit about how to do it well. Importantly, we've created a culture where dogfooding is expected and encouraged. We've also put into place the tools and processes that help us get good feedback and act on it, improving our product.
+
+Doing this yourself is a two step process:
+
+### 1. Create a culture of dogfooding
+
+A hot take: forcing your team to dogfood your product is a bad sign. This means there is a fundamental mismatch between the work your team wants to do and the product you are building. There are a few potential reasons for this:
+
+1. Your team isn't the ideal user for your product.
+2. Your product isn't good enough for your team to use.
+3. You haven't created the right culture to support dogfooding.
+
+The first two are major problems beyond the focus of this article (they require a rethinking of your entire strategy). The third is the most common and fixable.
+
+To create a culture of dogfooding, first, you need buy-in from the top. The responsibility to dogfood does not belong to one person or team. The entire organization needs to be bought into using the product, providing feedback, and continuously improving the product. 
+
+Everyone at PostHog, from the cofounders to the marketing team, uses PostHog and provides feedback on their usage.  
+
+![Tim dogfooding](https://res.cloudinary.com/dmukukwp6/image/upload/cofounder_ec648e21d3.png)
+
+For feedback is critical to the dogfooding process and to make people comfortable with giving feedback, you need trust.  Your team must trust each other to give and [receive feedback](/handbook/people/feedback). Assume positive intent, acknowledge it, and do with it what you want. This might be uncomfortable, but dogfooding doesn't work without it.
```

Comment:
```suggestion
Feedback is critical to the dogfooding process and to make people comfortable with giving feedback, you need trust.  Your team must trust each other to give and [receive feedback](/handbook/people/feedback). Assume positive intent, acknowledge it, and do with it what you want. This might be uncomfortable, but dogfooding doesn't work without it.
```